### PR TITLE
fix: timeline safe processing events for destinations

### PR DESCRIPTION
**Pull request by oscb**
-  Timeline won't shortcircuit destinations if one fails or returns undefined (each DestinationPlugin should be independent)
- Fixed Braze plugin key ('Appboy')

### CODE REVIEW

File: packages/core/src/plugin.ts [hunk 1]
```diff
@@ -155,7 +155,7 @@ export class DestinationPlugin extends EventPlugin {
 
   execute(event: SegmentEvent): SegmentEvent | undefined {
     if (!this.isEnabled(event)) {
-      return undefined;
+      return;
```
Comment (prayansh) [inline packages/core/src/plugin.ts:158]: whats the difference here?

**Review by prayansh — APPROVED** [review]
LGTM

File: packages/core/src/plugin.ts (hunk 1 above)
Comment (oscb) [reply packages/core/src/plugin.ts:158]: Nothing. Just for consistency.

**Pull request merged**